ops/_tracing/export.py

```diff
@@ -0,0 +1,272 @@
+# Copyright 2022 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+# file except in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+"""FIXME Docstring."""
+
+from __future__ import annotations
+
+import contextlib
+import logging
+import os
+import ssl
+import threading
+import time
+import urllib.error
+import urllib.request
+from pathlib import Path
+from typing import Sequence
+
+# FIXME: single-file Python package can't be marked as py.typed
+import otlp_json  # type: ignore
+from opentelemetry.instrumentation.urllib import URLLibInstrumentor
+from opentelemetry.sdk.resources import Resource
+from opentelemetry.sdk.trace import ReadableSpan, TracerProvider
+from opentelemetry.sdk.trace.export import BatchSpanProcessor, SpanExporter, SpanExportResult
+from opentelemetry.trace import get_tracer_provider, set_tracer_provider
+
+import ops
+import ops._tracing.buffer
```

Comment:
Relative `.` import?

Reply:
🤔 the split between `from .` and `from ops.` is 13 - 22.
I supposed I leaned towards the majority use.

In the end both work just fine, happy to change.

Reply:
This would probably be a good addition to our style guide (covering at least ops and ops-scenario, but potentially also Jubilant and others). And following a decision, a PR that normalises everything to match.

Reply:
Yes, that's a good idea. Let's discuss briefly in today's daily, and then one of us can update the style guide and open a PR.